client/src/admin/MainSection/UserList/User/UserDetails/UserDetails.jsx

```diff
@@ -0,0 +1,39 @@
+import { useDispatch, useSelector } from 'react-redux';
+import './styles.scss';
+import { getOrders } from '../../../../../redux/slices/ordersSlice';
+import { useEffect } from 'react';
+import UserOrder from '../UserOrder/UserOrder';
+
+const UserDetails = ({ lastName, firstName, email, phoneNumber, address, id }) => {
+  const { orders } = useSelector(state => state.orders);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getOrders(id));
+  }, []);
+
+  const userOrders = orders.map(order => {
+    return {id: order.id, orders: order.orders};
```

Comment:
```suggestion
  const userOrders = orders.map(({ id, orders }) => ({ id, orders })
```
Nhưng sao phải cần tới cái này nhỉ
Còn những trường khác ngoài id với orders ở bên trong mảng orders à anh?

Reply:
ừa vì a thấy còn trường userId và createdAt nữa nên a mới map lọc nó ra.

Reply:
ừa vì a thấy còn trường userId và createdAt nữa nên a mới map lọc nó ra.

Reply:
ừa vì a thấy còn trường userId và createdAt nữa nên a mới map lọc nó ra.